hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/FilteringMessageReceiver.java

```diff
@@ -42,11 +43,19 @@ public FilteringMessageReceiver(MessageReceiver receiver,
     }
 
     private boolean allow(Message message) {
-        FilterResult result = filterChain.apply(message);
+        FilterResult result = filterChain.apply(getMessageContent(message));
         filteredMessageHandler.handle(result, message, subscription, (offset) -> receiver.commit(singleton(offset)));
         return !result.isFiltered();
     }
 
+    private MessageContent getMessageContent(Message message) {
```

Comment:
It would be nice, but **Message** (from *consumers module*) is not available in *common module*.